test/integration/ember-i18n-csv-test.js

```diff
@@ -56,6 +64,23 @@ describe('integration - ember-i18n-csv', function() {
         });
       });
     });
+    describe('merge', function() {
+      beforeEach(function() {
+        fs.copySync('test/fixtures/locales', 'tmp/locales', { clobber: true });
+      });
+
+      afterEach(function() {
+        fs.emptyDirSync('tmp');
+      });
+
+      it('handles option --merge', function() {
+        return emberI18nCsv('to-js', 'tmp/locales', 'test/fixtures/i18n-merge.csv', { merge: true }).then(() => {
+          return areDirsEqual('tmp/locales', 'test/fixtures/locales').then(areSame => {
```

Comment:
Yea, then I think that new test can replace this test, because it will cover the merge protection and the additional properties.